documentation/modules/post/windows/gather/bitlocker_fvek.md

```diff
@@ -0,0 +1,65 @@
+## Vulnerable Application
+
+This module enumerates ways to decrypt Bitlocker volume and if a recovery key is stored locally or can be generated, dump the Bitlocker master key (FVEK)
+
+## Verification Steps
+  1. Start msfconsole
+  2. Get meterpreter session
+  3. Do: ```use post/windows/gather/bitlocker_fvek```
+  4. Do: ```set SESSION <session id>```
+  5. Do: ```set DRIVE_LETTER <letter>```
+  6. Do: ```run```
+
+## Options
+
+  ***
+  DRIVE_LETTER
+  ***
+  Dump information from the DRIVE_LETTER encrypted with Bitlocker.
+
+  ***
+  RECOVERY_KEY
+  ***
+  Use the recovery key provided to decrypt the Bitlocker master key (FVEK).
```

Comment:
is there a default?

Reply:
There is no default. This is a user provided value where if the the recovery key is known, it can be provided, else the module attempts to locate the recovery key on the volume.